models/articles.model.js

```diff
@@ -27,4 +24,18 @@ exports.fetchArticlesById = (article_id) => {
     
     })
 
+}
+
+exports.fetchCommentsByArticleId = (article_id) => {
+    
+    return db.query(`SELECT * FROM comments WHERE article_id = $1  ORDER BY comments.created_at DESC;`,[article_id]).then((result) => {
+        const comments = result.rows
+        if (result.rowCount === 0) {
```

Comment:
To pass the suggested test, you will need to check why the rows are empty. Is it because the article doesn't exist or is it because the article exists but has no comments?